### DISCUSSION TITLE
Welcome to req Discussions!

### DISCUSSION THREAD

**Discussion by imroc** [imroc/req#154] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Xunzhuo** [comment]
Thanks for starting this awesome project, it is pretty cool! But for the community side, do we have any roadmap or future plan of Req? This is good for newcomers to contribute.

**Reply by imroc** [reply]
Right here: https://github.com/users/imroc/projects/1/views/2

There are no major features to be developed at the moment.

**Reply by Xunzhuo** [reply]
Got it, thanks:) Maybe I should go through all the abilities of req currently and see if there are any bugs to fix or user experience issues to improve.

**Reply by imroc** [reply]
Thanks a lot, you can go to official website to learn usage and examples: https://req.cool
